docs/block-api.md

```diff
@@ -52,22 +52,34 @@ description: 'Block showing a Book card.'
 
 #### Category
 
-* **Type:** `String [ common | formatting | layout | widgets | embed ]`
+* **Type:** `String`
+
+Blocks are grouped into categories to help users browse and discover them. 
 
-Blocks are grouped into categories to help users browse and discover them. The core provided categories are `common`, `formatting`, `layout`, `widgets`, and `embed`.
+The core provided categories are:
+* common
+* formatting
+* layout
+* widgets
+* embed
 
 ```js
 // Assigning to the 'widgets' category
 category: 'widgets',
 ```
 
+Plugins & Themes can also register [custom block categories](../docs/block-api/extensibility.md).
+
 #### Icon (optional)
 
 An icon property should be specified to make it easier to identify a block. These can be any of [WordPress' Dashicons](https://developer.wordpress.org/resource/dashicons/), or a custom `svg` element.
 
 ```js
 // Specifying a dashicon for the block
 icon: 'book-alt',
+
+// Specifying a custom svg for the block
+icon: <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="none" d="M0 0h24v24H0V0z" /><path d="M19 13H5v-2h14v2z" /></svg>,
```

Comment:
Until #9372 is fixed these should at least follow the style of #9269 and include the needed attributes for accessibility.